solr/core/src/java/org/apache/solr/cloud/ReplicateFromLeader.java

```diff
@@ -76,12 +76,12 @@ public void startReplication(boolean switchTransactionLog) throws InterruptedExc
       }
       log.info("Will start replication from leader with poll interval: {}", pollIntervalStr );
 
-      NamedList<Object> slaveConfig = new NamedList<>();
-      slaveConfig.add("fetchFromLeader", Boolean.TRUE);
-      slaveConfig.add(ReplicationHandler.SKIP_COMMIT_ON_MASTER_VERSION_ZERO, switchTransactionLog);
-      slaveConfig.add("pollInterval", pollIntervalStr);
+      NamedList<Object> followerConfig = new NamedList<>();
+      followerConfig.add("fetchFromLeader", Boolean.TRUE);
+      followerConfig.add(ReplicationHandler.SKIP_COMMIT_ON_LEADER_VERSION_ZERO, switchTransactionLog);
+      followerConfig.add("pollInterval", pollIntervalStr);
       NamedList<Object> replicationConfig = new NamedList<>();
-      replicationConfig.add("slave", slaveConfig);
+      replicationConfig.add("follower", followerConfig);
```

Comment:
I left this intentionally using the new parameters because this configuration is only read locally

Reply:
🤨

Reply:
left intentionally — what are you saying?

Reply:
It should be left intentionally. It comes from the leader, right?

Reply:
For 8.x branches, every time we set a parameter or configuration (for requests, etc) we use the legacy names. In this particular case, I'm using the new parameters even for setting, and the reason is that this is a configuration that is set and read locally only. The TLOG/PULL replicas create this to create their replication process.